src/components/Header.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable jsx-a11y/alt-text */
+import React, { useContext } from 'react'
+import {ProductContxt} from '../App'
+import '../style/Header.css' 
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCartShopping } from "@fortawesome/free-solid-svg-icons"
+
+function IamgeCombonant() {
```

Comment:
Very bad spelling, and why did you create a sperate component for just a single div?

Reply:
done